src/status_im/contexts/wallet/send/utils.cljs

```diff
@@ -47,3 +65,104 @@
                (assoc acc k (if (money/equal-to v 0) "<0.01" v)))
              {}
              amounts))
+
+(defn token-valid-networks
+  [{:keys [balances-per-chain disabled-chain-ids]}]
+  (let [disabled-set (set disabled-chain-ids)]
+    (->> balances-per-chain
+         (filter (fn [[_ {:keys [chain-id]}]]
+                   (not (contains? disabled-set chain-id))))
+         (map first))))
+
+(def ^:private network-priority-score
+  {:ethereum 1
+   :optimism 2
+   :arbitrum 3})
+
+(def ^:private available-networks-count
+  (count (set (keys network-priority-score))))
+
+(defn clean-network-amounts
```

Comment:
I would suggest some other naming here. "cleaning" amounts make me think about laundering)) In fact the function resets the loading state. So it can be something like `remove-loading-state`. Maybe not ideal but a bit closer to what happens inside the function)

Reply:
Updated to `reset-network-amounts-to-zero`